src/Validation/Validator.php

```diff
@@ -260,13 +260,13 @@ public function validateParams(array $formDetails, bool $strictValidation = true
 						break;
 					// Check validation for min number value.
 					case 'min':
-						if ((string) $dataValue > (string) $inputValue) {
+						if ((string) $dataValue > (string) $inputValue && !empty($inputValue)) {
```

Comment:
assume min number value is `42` and submission value is `112` - I think this would be false?

Reply:
this is tested and it work